docs/boxplot/BoxPlot.demo.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import {
+  AnnotationBoxPlotProps,
+  Plot,
+  Annotations,
+  Annotation,
+  SeriesPoint,
+} from 'react-plot';
+import { ScatterSeries, Axis } from 'react-plot';
+
+import random from 'random/dist/cjs';
```

Comment:
```suggestion
import random from 'random';
```

Reply:
If you want seeded random numbers please use our package: https://github.com/mljs/xsadd

Reply:
xsadd doesn't provide a way to generate numbers using a normal distribution